node/tests/utils.rs

```diff
@@ -346,33 +353,44 @@ impl MASQNode {
         } else {
             node_home_directory("integration", test_name)
         };
+        let chain = Self::get_chain_from_config(&config_opt);
         if sterile_logfile {
             let _ = Self::remove_logfile(&data_dir);
         }
         let ui_port = Self::ui_port_from_config_opt(&config_opt);
         let mut command = command_getter(&data_dir, config_opt, sterile_database);
-        eprintln!("{:?}", command);
```

Comment:
This eprintln!() was not from you. It'd been there for months. Please put it back. 
You can also extend the message if you want.
For example:
`eprintln!("Launching MASQNode with this command: {:?}", command)`;

Reply:
I put that back